hmpo-lev/email/text/email-verification.ftl

```diff
@@ -0,0 +1 @@
+${msg("emailVerificationBody",linkExpirationFormatter(linkExpiration), user.firstName, user.lastName, link)?no_esc}
```

Comment:
Can this `msg` function be used to encapsulate other sections of text? i.e. could you add the following to the `message_en.properties` file?:
```
serviceLinks=Births: http://lev.homeoffice.gov.uk\n\n\
Deaths: http://lev.homeoffice.gov.uk/death\n\n\
Marriages: https://lev.homeoffice.gov.uk/marriage\n\n\
Civil partnerships: https://lev.homeoffice.gov.uk/partnership\n\n
serviceLinksHtml=<ul><li>Births: <a href="http://lev.homeoffice.gov.uk">http://lev.homeoffice.gov.uk</a></li>\
<li>Deaths: <a href="http://lev.homeoffice.gov.uk/death">http://lev.homeoffice.gov.uk/death</a></li>\
<li>Marriages: <a href="https://lev.homeoffice.gov.uk/marriage">https://lev.homeoffice.gov.uk/marriage</a></li>\
<li>Civil partnerships: <a href="https://lev.homeoffice.gov.uk/partnership">https://lev.homeoffice.gov.uk/partnership</a></li></ul>
```
That way you could make the template more flexible, for use by other projects.

Reply:
They can't be added to the message_en.properties file but I created a utils file for constants/variables that could be helpful down the line...in this case it did mean I could remove the duplication of text from the message_en.properties file and then import it in the messages that required.

Reply:
Ok cool, that sounds like an improvement